kie-camel/src/main/java/org/kie/camel/KieCamelConstants.java

```diff
@@ -18,12 +18,12 @@
 
 public class KieCamelConstants {
 
-    public static final String KIE_HEADERS_PREFIX = "kie.";
+    public static final String KIE_HEADERS_PREFIX = "CamelKie";
```

Comment:
Consistent with the other Camel components "CamelInfinispan..." or "CamelJdbc..."